submission/ContextTabs/popup.js

```diff
@@ -0,0 +1,73 @@
+document.addEventListener('DOMContentLoaded', () => {
+    console.log("Popup script loaded");
+
+    function loadContexts() {
+        chrome.storage.local.get(null, function(result) {

```

Comment:
how about you use the `extensionAPI` here as well to be cross-browser compatible?